.gitignore

```diff
@@ -83,3 +83,4 @@ pubspec\.lock
 \.flutter-plugins-dependencies
 ios/Podfile.lock
 dartesr/
+ios/Flutter/Flutter.podspec
```

Comment:
Note: This file showed up with a warning to NOT check it in - so added to ignore